src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -54,21 +56,32 @@ def get(self, request, organization, trace_id):
         except NoProjects:
             return Response(status=404)
 
+        detailed = request.GET.get("detailed", "0") == "1"
+        event_id = request.GET.get("event_id")
+
+        # selected_columns is a set list, since we only want to include the minimum to render the trace
+        selected_columns = [
+            "id",
+            "timestamp",
+            "transaction.duration",
+            "transaction.op",
```

Comment:
Minor, but we can probably move `transaction.op` under the detail parameter as well. (need to update frontend type to correspond with this).

Reply:
:P Way ahead of you: https://github.com/getsentry/sentry/pull/24544/files#diff-619b86f549a9f11a566ca757be8da43a63bcaf6e4d7d3fe88a4222937a0209c6R77

I'll do it as cleanup once we update the frontend.